src/app/visual/lookup/lookup-visual.component.ts

```diff
@@ -67,17 +67,17 @@ export class LookupVisualComponent implements OnInit {
   ];
 
   public friends: any[] = [
-    { id: 16, name: 'Susan' },
```

Comment:
This was not a part of this change but was a bug in our visual tests that I found during this work. We weren't using direct object references which was causing issues where the items were not selected when the show more modal opened.